operators/src/engine/operator_impl.rs

```diff
@@ -102,7 +102,7 @@ impl<P, R, S> InitializedOperatorImpl<P, R, S> {
     }
 }
 
-impl<P, R, S> InitializedOperator for InitializedOperatorImpl<P, R, S>
+impl<P, R, S> InitializedOperatorBase for InitializedOperatorImpl<P, R, S>
 where
     P: std::fmt::Debug + Clone + 'static,
     R: std::fmt::Debug + Clone + 'static + ResultDescriptor,
```

Comment:
This is also strange, again, Debug and Clone should be on some other types.

Reply:
there are no other types involved here. Different Parameters and State have nothing in common by now. I simplified the bounds though: 51e1381cde7d2c2f6080c710fb06190362843ef2